apps/next/src/content/docs/llamaindex/modules/data/readers/index.mdx

```diff
@@ -32,14 +32,6 @@ import { HTMLReader } from '@llamaindex/readers/html'
 
 Additionally the following loaders exist without separate documentation:
```

Comment:
remove this and update `import` list above to contain all readers from readers package